README.md

```diff
@@ -80,16 +68,6 @@ bin/rake app:jasmine
 bin/rake app:jasmine:ci
 ```
 
-### Viewing the component guide
-
-View the component guide in your browser with:
-
-```sh
-./startup.sh
-```
-
-The guide should be available at http://localhost:3212
```

Comment:
Isn't it helpful to know how to access the locally running instance of this?

Reply:
This is printed out when you run the script, so I think it's OK to remove it - this matches the approach we've taken in other frontend repos.

Reply:
I personally can't think of a reason why it would be beneficial to remove this line over making it crystal clear where to go to get the component guide.

Reply:
Hey @owenatgov, thanks for taking a look. I'm a bit confused: can you explain what you mean a bit more? What scenario are you thinking about in your comment?

Reply:
No worries! So a lot of this is a bit conjecture-y but: new devs learning to use our tooling will arrive at the docs first if they get confused. The script in question in this case does make it clear where to go but terminal environments aren't the most usable places and what'll probably end up happening is that any devs who miss that line in the terminal will ask where to go, realise through some discussion that it's in the terminal window and feel silly. We don't really want to be perpetuating imposter syndrome (even in a small way)

I feel this way about quite a few changes in this PR which Andy and Alex have already outlined. I understand why you've made them but similarly, in my opinion, consistency and "tidiness" gain us very little in terms of documenting our process over spelling things out clearly to devs. There is zero harm (again IMO) in making our docs "dumb".

Reply:
Thanks for explaining @owenatgov, that makes sense to me. Even if someone does see the output, it's also possible they may not understand what it means / whether it's related to their goal.

I do think we need to be careful with putting too much "stuff" in our READMEs:

- Reading a long document is off-putting, and can equally contribute to imposter syndrome, if someone feels they have a mountain to learn before they can do anything. One line is obviously fine, and this one is also relevant to the main goal of a README, which is to provide the minimum info to get started with the repo (vs. doing everything ever).

- Maintaining a long document is also off-putting. It's easier for false information to persist this way. In general, we avoid reviewing READMEs on GOV.UK, and most of them contained out-of-date or misleading info as a consequence. So we should think hard about what's necessary, and whether we can centralise the knowledge.

Although this thread is about one line, it's actually the whole section that I'm suggesting we delete. The reason for this is that the content is misleading, because it only applies to one way of running this repo: if you're using GOV.UK Docker, these instructions won't work. What I've done instead is to point at [the central dev doc about this](https://github.com/alphagov/govuk_publishing_components/pull/2060/commits/aa7860921635ddb7233b9af8abea6b5842eba66b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R50):

- The doc explains both alternatives for running this and other frontend repos.
- The doc also covers how to view the index page, which for this app, is the component guide.

We [agreed this approach for Collections](https://github.com/alphagov/collections/pull/2375), with the aim of rolling it out across other frontend repos. I went to some effort to ask frontend devs to give their thoughts at that stage, so I'm surprised to be encountering comments now. Equally I want to make sure we're making a change we can all agree with. 

That was quite a long comment, sorry! I've been pondering how best to proceed with this PR, so it's more a collection of my thoughts. Some questions I'd like you to consider are:

- Is the central doc on frontend development sufficient to replace this section?
- If it's not, what do we need to have here? Do we need this in the other frontend repos as well?
- How can we support people who don't use the startup.sh scripts at the same time?

Reply:
You make some compelling point here. I one little reflection here which you might find beneficial:

>We agreed this approach for Collections, with the aim of rolling it out across other frontend repos. I went to some effort to ask frontend devs to give their thoughts at that stage, so I'm surprised to be encountering comments now.

A hypothesis about this specifically is that I had no idea you could run this repo using govuk-docker. In my mind, and I think the minds of a lot of other FEs on govuk, this was a separate because it's a gem and the docs element lives in a different world to our frontend apps. It sort of does but also that's no reason not to use govuk-docker really. I therefore compartmentalise it differently and made the (incorrect) assumption that this was the only way to run it. This probably speaks to your point that these docs are in fact misleading.

I saw you asking for FE input re: dockerising everything and didn't get involved as I was already using govuk-docker for everything... except this.

As for the original comment, I frankly don't know enough about how rails does serving to make a call here eg: I'm not sure if the rest of our apps, if run using the `startup.sh` method, would use `localhost:3212` or something else. I think I'd defer to @alex-ju's comments on this one ie: if they've been agreed and there's been some sort of investigation then I retract my comment.